### ISSUE TITLE
The Script says AccountSid not set

### ISSUE THREAD

**Issue by kakashikuna** [ericallenpaul/Zabbix-Twilio.sh#6]
The Script says AccountSid not set but i added all Data in the config File

`AccountSid not set, it must be provided in the config file, or on the command line.
Usage: twilio-sms [-v] [-c configfile] [-d callerid] [-u accountsid] [-p authtoken] number [number[number[...]]]`

**Comment by hriday** [comment]
Explicitly set the config file location in the script: 

CONFIGFILE=/usr/lib/zabbix/alertscripts/twilio-sms.conf

**Comment by kakashikuna** [comment]
Yes AccountSID, AuthToken and Callerid.

Have also done this

> Explicitly set the config file location in the script:
> 
> CONFIGFILE=/usr/lib/zabbix/alertscripts/twilio-sms.conf

But in Zabbix the same Error

**Comment by hriday** [comment]
Does it work from the command line? Check the permissions on the files.

**Comment by kakashikuna** [comment]
echo "TEST" | /usr/lib/zabbix/alertscripts/twilio-sms.sh -c /usr/lib/zabbix/alertscripts/twilio-sms.conf +4XXXXXXXXXX
Sending SMS to +4XXXXXXX from +1XXXXXXXXX...ok

that works it

**Comment by hriday** [comment]
Right. When you explicitly set the config file in the script, you avoid the
need to call it with -c in zabbix. I was able to avoid the error you are
getting by doing that.

**Comment by kakashikuna** [comment]
Zabbix Settings

Name -> Twilio
Type -> Script
Scriptname -> twilio-sms.sh

**Comment by kakashikuna** [comment]
Here a screenshot from my Zabbix Settings
![Zabbix Settings](https://user-images.githubusercontent.com/26281803/228577270-d84759eb-b0a2-4ae1-8f68-49a42f3333a9.png)
